uav1:
  home:
    x: 4.5
    y: -1.0

uav2:
  home:
    x: 4.5
    y: 1.0

height: 4.2

corners:
  p1:
    x: 3
    y: -3
  p2:
    x: 3
    y: 3
  p3:
    x: -3
    y: 3
  p4:
    x: -3
    y: -3

segment_size: 4